projects/mmf_transformer/configs/vqa2/defaults.yaml

```diff
@@ -45,4 +45,6 @@ training:
 
 checkpoint:
   pretrained_state_mapping:
-    transformer: transformer
+    pooler: pooler
+    backend.transformer: backend.transformer
```

Comment:
Won't `backend: backend` work?

Reply:
It will work. However I want to keep them separate as we don't always want to initialize the embeddings from the pretrained models (due to different number of modalities between pretrained and finetuned tasks).